plugins/UserCountry/tests/System/AttributeHistoricalDataWithLocationsTest.php

```diff
@@ -101,6 +103,12 @@ public function testExecute_ShouldReturnLogAfterWorkingWithSomeData()
             'period'  => 'month'
         );
 
+        // we need to manually reload the translations since they get reset for some reason in IntegrationTestCase::tearDown();
+        // if we do not load translations, a DataTable\Map containing multiple periods will contain only one DataTable having
+        // the label `General_DateRangeFromTo` instead of many like `From 2010-01-04 to 2010-01-11`, ' `From 2010-01-11 to 2010-01-18`
+        // As those data tables would all have the same prettyfied period label they would overwrite each other.
+        Translate::loadAllTranslations();
```

Comment:
This is caused by `Fixture::clearInMemoryCaches()`, perhaps adding the call to Translate would be a cleaner? Or maybe `Translate::reset()` shouldn't be called in the clearInMemoryCaches...